### ISSUE TITLE
mate-volume-control: lack of Russian localization for "Front Left" and "Front Right"

### ISSUE THREAD

**Issue by HectorB-2020** [mate-desktop/mate-desktop#498]
#### Expected behaviour
"**Hardware**", "**Test Speakers**"
_Front Left_ and _Front Right_ must be translated to Russian

#### Actual behaviour
"**Hardware**", "**Test Speakers**"
_Front Left_ and _Front Right_ are left in English even with Russian region and localization

#### Steps to reproduce the behaviour
See about

#### MATE general version
```
mate-about --version
MATE Desktop Environment 1.24.0
```

#### Package version
```
mate-volume-control --version
mate-volume-control 1.24.0

```
#### Linux Distribution
Ubuntu 20.04 LTS latest
Alt Workstation P9.2

**Comment by lukefromdc** [comment]
By this do you mean the text in and above the buttons under the speaker images in the speaker testing dialog, or the spoken words that sound when the "test" button is clicked?

**Comment by HectorB-2020** [comment]
Correct, I mean the text "Front Left" and "Front Right" in the dialog.

**Comment by HectorB-2020** [comment]
BTW, I've just confirmed that this is still not translated to Russian in MATE 1.26 based on the latest Ubuntu 21.10.

**Comment by N0rbert** [comment]
@HectorB-2020, сколько можно!? Вам четко сказали, что переводы выполняются на сервисе Transifex:

> https://github.com/mate-desktop/caja/issues/1590#issuecomment-995218118 
>Again, we're using transifex for our translations. We can't fix it here !
>Please become a member of your language team at transifex and fix translations there.
>https://www.transifex.com/mate/MATE/languages/

Пожалуйста, зарегистрируйтесь на сервисе transifex и внесите свой вклад там!
Пожалуйста, прекратите захламлять трекеры ошибок проблемами с переводом!

---

@HectorB-2020 how long!? It was told to use Transifex service for the translations.

Please register on transifex service and make your impact there!
Please stop filling issues about translations!
